### PR TITLE
Adding Pagination to Supervisor Dashboard

### PR DESCRIPTION
**Pull request by AK56**
## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

The story: https://osi-cwds.atlassian.net/browse/CANS-750

supervisordashboardpagination

### CODE REVIEW

**Comment by wmitchell-cambria** [comment]
Could you please add a link to the story, and perhaps also a screenshot of the pagination?

**Comment by AK56** [comment]
jenkins retest please

**Comment by AK56** [comment]
jenkins retest this please

File: app/javascript/Application/components/Staff/StaffTable.js [hunk 1]
```diff
@@ -37,15 +37,14 @@ const columns = [
...
-const MAX_SUBORDINATES = 1000
 const MIN_ROWS = 3
 
 const StaffTable = ({ staff }) => (
   <DataGrid
     data={staff}
     columns={columns}
-    showPagination={false}
-    defaultPageSize={MAX_SUBORDINATES}
+    pageSizeOptions={[10, 25, 50]}
```
[the remark this answers is not in the document]
Comment (AK56) [reply app/javascript/Application/components/Staff/StaffTable.js]: possibly another constant would be nice

**Pull request merged**